### ISSUE TITLE
TrayContentProvider does not found when using Gradle 1.5.0-beta2 & proguard enabled

### ISSUE THREAD

**Issue by yshrsmz** [grandcentrix/tray#57]
Though I've just reported the issue in [Google's issue tracker](https://code.google.com/p/android/issues/detail?id=193705&thanks=193705&ts=1447124312), I thought it would be good to also report here.

I just got following error when tried to use Android Gradle plugin 1.5.0-beta2 with proguard enabled.
without proguard, it works fine. but when I enable proguard, it starts crashing right after I launch the app.
With Gradle plugin 1.3.0, it works fine.

I'm using 1.0.0-rc3 of the tray.

```
java.lang.RuntimeException: Unable to get provider net.grandcentrix.tray.provider.TrayContentProvider: java.lang.ClassNotFoundException: Didn't find class "net.grandcentrix.tray.provider.TrayContentProvider" on path: DexPathList[[zip file "/data/app/net.yslibrary.omnitweety-1/base.apk"],nativeLibraryDirectories=[/data/app/net.yslibrary.omnitweety-1/lib/arm64, /data/app/net.yslibrary.omnitweety-1/base.apk!/lib/arm64-v8a, /vendor/lib64, /system/lib64]]
                                                     at android.app.ActivityThread.installProvider(ActivityThread.java:5156)
                                                     at android.app.ActivityThread.installContentProviders(ActivityThread.java:4748)
                                                     at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4688)
                                                     at android.app.ActivityThread.-wrap1(ActivityThread.java)
                                                     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1405)
                                                     at android.os.Handler.dispatchMessage(Handler.java:102)
                                                     at android.os.Looper.loop(Looper.java:148)
                                                     at android.app.ActivityThread.main(ActivityThread.java:5417)
                                                     at java.lang.reflect.Method.invoke(Native Method)
                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
                                                  Caused by: java.lang.ClassNotFoundException: Didn't find class "net.grandcentrix.tray.provider.TrayContentProvider" on path: DexPathList[[zip file "/data/app/net.yslibrary.omnitweety-1/base.apk"],nativeLibraryDirectories=[/data/app/net.yslibrary.omnitweety-1/lib/arm64, /data/app/net.yslibrary.omnitweety-1/base.apk!/lib/arm64-v8a, /vendor/lib64, /system/lib64]]
                                                     at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
                                                     at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
                                                     at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
                                                     at android.app.ActivityThread.installProvider(ActivityThread.java:5141)
                                                     at android.app.ActivityThread.installContentProviders(ActivityThread.java:4748) 
                                                     at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4688) 
                                                     at android.app.ActivityThread.-wrap1(ActivityThread.java) 
                                                     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1405) 
                                                     at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                     at android.os.Looper.loop(Looper.java:148) 
                                                     at android.app.ActivityThread.main(ActivityThread.java:5417) 
                                                     at java.lang.reflect.Method.invoke(Native Method) 
                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
                                                 	Suppressed: java.lang.ClassNotFoundException: net.grandcentrix.tray.provider.TrayContentProvider
                                                     at java.lang.Class.classForName(Native Method)
                                                     at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
                                                     at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
                                                     at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
                                                     		... 12 more
                                                  Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack trace available
```

**Comment by passsy** [comment]
have you added this to your proguard rules?

```
-keep public class * extends android.content.ContentProvider
```

**Comment by yshrsmz** [comment]
No, but I tried with this, and same crash

```
-keep class net.grandcentrix.tray.** { *; }
```

**Comment by passsy** [comment]
solved in 1.5.0

**Comment by WangLaoSeng** [comment]
i have met this problem too

it's because multidex ,make sure that you app's multidex is compatible with android under 5.0
